composer.json

```diff
@@ -64,6 +64,7 @@
         "twig/twig": "^2.14 || ^3.0"
     },
     "require-dev": {
+        "async-aws/simple-s3" : "^1.1",
```

Comment:
Can we support `^1.0` ?
We also have to add "aws/aws-sdk-php": "<1.0", in the conflict section.

Reply:
Also take a look at the failing build for the composer linter

Reply:
@VincentLanglet are you sure about "aws/aws-sdk-php": "<1.0" conflict? There is already "aws/aws-sdk-php": "<3.0" in the conflict section, and probably it's there for a reason?

Reply:
I meant `async-aws/simple-s3 < 1.0` sorry

Reply:
Sure, makes sense. Added it.